saleor/graphql/checkout/types.py

```diff
@@ -176,6 +176,12 @@ def resolve_available_shipping_methods(root: models.Checkout, info):
         )
         if available is None:
             return []
+        if display_gross_prices():
```

Comment:
Hhm I think that here we should confirm more condition. We have 3 settings which can influence on shipping and taxes.
```

def include_taxes_in_prices():
    return Site.objects.get_current().settings.include_taxes_in_prices


def display_gross_prices():
    return Site.objects.get_current().settings.display_gross_prices


def charge_taxes_on_shipping():
    return Site.objects.get_current().settings.charge_taxes_on_shipping
```

Reply:
`ShippingMethodChoiceField` form doesn't check this also. I assume that it can be wrong. Please confirm that.

Reply:
sure, I'll check this